packages/manager/modules/iam/src/components/conditions/conditionList/conditionList.controller.js

```diff
@@ -4,6 +4,12 @@ export default class IAMConditionListController {
   $onInit() {
     this.condition = null;
     this.closeModal();
+
+    this.pagination = {
+      currentPage: 1,
```

Comment:
As you've created an object to manage the pagination, you can move the init of page size here.
```suggestion
      currentPage: 1,
      pageSize: 5,
```